include/RAJA/policy/openmp/atomic.hpp

```diff
@@ -91,7 +111,7 @@ RAJA_INLINE T atomicInc(omp_atomic, T volatile *acc)
 #pragma omp atomic capture
   {
     ret = *acc;  // capture old for return value
-    *acc += 1;
+    *acc += (T)1;
```

Comment:
Is there a preferred style of casting in cases like this?

Reply:
I don't know that we've settled on one, I've become fond of `T(1)` for its brevity, but I think we also use `static_cast<t>(1)`.

Reply:
I switched to T(1) since it is shorter than the static_cast alternative.